docs/variants/protectli_vp2420/hardware-matrix.md

```diff
@@ -0,0 +1,35 @@
+# Hardware configuration matrix
+
+## Introduction
+
+This document describes the hardware configuration used for validation of the
+coreboot port on the Protectli VP2420 firewall.
+
+## Protectli VP2420
+
+| Component              | Description                                              |
+|------------------------|----------------------------------------------------------|
+| **CPU**                | Intel(R) Celeron(R) J6412 @ 2.60GHz                      |
+| **RAM**                | CRUCIAL CT4G4SFS824A                                     |
+|                        | SAMSUNG M471A4G43MB1-CTD                                 |
+|                        | SAMSUNG M471A1K43CB1-CTD                                 |
+|                        | SAMSUNG M471A5244BB0-CRC                                 |
+|                        | KINGSTON KVR24S17S8/8                                    |
+|                        | KINGSTON KVR26S19S8/16                                   |
+| **Flash memory**       |                                                          |
```

Comment:
Information about `Flash memory` should be added

Reply:
@sulewskiprzemyslaw I've added the name for `vp2420`. I can also add it for `vp2410`, but someone needs to give me chip name. I'm guessing that it's the same chip, but someone should confirm that.

Reply:
@Stojek139808 you may find it out by performing the flash operation on VP2410 in lab from RTE. flashrom will output chip model (may not be full chip model thought). Alternatively you may make a quick photo and decode it. If the info was not available yet, nobody has such info most likely.

Reply:
@Stojek139808 Lets' add the flash chip for VP2410 too: `Macronix MX25U6473F`. With that we could merge this PR I think

Reply:
Done.